docs/demomaterials/README.md

```diff
@@ -0,0 +1,145 @@
+_Table of contents_
```

Comment:
do you have a link to this new page? If not, please add one.

Reply:
@dankenigsberg which page? to the table of contents?